lib/SILGen/SILGen.cpp

```diff
@@ -1284,22 +1302,22 @@ void SILGenModule::emitDestructor(ClassDecl *cd, DestructorDecl *dd) {
   if (dd->hasBody()) {
     SILDeclRef destroyer(dd, SILDeclRef::Kind::Destroyer);
     SILFunction *f = getFunction(destroyer, ForDefinition);
-    preEmitFunction(destroyer, dd, f, dd);
+    RegularLocation loc(dd);
```

Comment:
This change seems like NFC here (which would be correct). Just making sure, is that intentional?

Reply:
I did it to match the code all around to it.